locations/hours.py

```diff
@@ -951,10 +951,39 @@ def add_range(self, day, open_time, close_time, time_format="%H:%M", closed=CLOS
         self.days_closed.discard(day)
         self.day_hours[day].add((open_time, close_time))
 
+    @staticmethod
+    def time_struct_instance(hour, minutes):
+        return time.struct_time((1900, 1, 1, hour, minutes, 0, 0, 1, -1))
```

Comment:
I cannot believe there is no nicer way to do this :(

I guess I can go through string?

Reply:
Code directly above actually uses strptime so maybe it should be treated as preferred method?